packages/circuit-ui/components/Notification/Notification.tsx

```diff
@@ -59,9 +55,9 @@ const colorMap = {
 } as const;
 
 const iconMap = {
-  success: CircleCheckmarkFilled,
-  error: CircleCrossFilled,
-  warning: CircleWarningFilled,
+  success: Confirm,
+  error: Alert,
+  warning: Notify,
```

Comment:
We should align these variant names in the new notification components.

Reply:
Excellent point! Wonder if v4 should also rename them in the legacy notification components?

Reply:
Maybe, up to you.

Reply:
In the end I'd prefer keeping this for the new notification components